packages/plugin-oas/src/SchemaGenerator.ts

```diff
@@ -560,6 +560,12 @@ export abstract class SchemaGenerator<
         type: 'type',
       })
 
+      const nullableEnum = schema.enum.includes(null);
+      if (nullableEnum) {
+        baseItems.push({ keyword: schemaKeywords.nullable })
+      }
+      const filteredValues = schema.enum.filter((value) => value !== null);
+
       // x-enumNames has priority
       const extensionEnums = ['x-enumNames', 'x-enum-varnames']
```

Comment:
I wasn't sure if I should use the filtered values in this case. Any thoughts on how to handle this?